### ISSUE TITLE
Dropout is not implemented yet

### ISSUE THREAD

**Issue by akshay951228** [tensorflow/tfjs-core#659]
I'm importing tensor model weight in DLJs, in my model I have dropout but dropout is not implemented yet.So my question is when it will be implemented

**Comment by dsmilkov** [comment]
Dropout is a great example for an external contribution. Happy to provide review!

**Comment by nsthorat** [comment]
@caisq is this something that would be easy to push into deeplearnjs?

**Comment by caisq** [comment]
@nsthorat Feel free to assign this FR to me.

**Comment by dsmilkov** [comment]
Awesome. Trying to assign you but looks like we need to give you write access - will do now

**Comment by abrad1212** [comment]
Couldn't you implement Dropout with everything in the current API?

Please see [Tensorflow's implementation](https://github.com/tensorflow/tensorflow/blob/r1.5/tensorflow/python/ops/nn_ops.py#L2135-L2196)

**Comment by caisq** [comment]
@abrad212 thanks for the ping. I have this on my TODO list

**Comment by abrad1212** [comment]
@caisq No problem. Anything to help 😄

**Comment by abrad1212** [comment]
@caisq I'm traveling today, and I think I have time over this weekend to try to implement DropOut. Hopefully, the implementation will be similar to the link I posted above, and translate easy. I'll let you know if I make any progress.

**Comment by acostin1** [comment]
Would love to test it, I think I need to use dropout layers because my model is over-fitting too fast.

**Comment by kwhitley** [comment]
Agreed re. all... definitely need dropout implementation stat.  The basic models without dropout, regularization, noise, etc overfit like a champ.  

😞

**Comment by nsthorat** [comment]
Coming soon, I promise!

**Comment by nsthorat** [comment]
This issue was moved to tensorflow/tfjs#117
